react/features/welcome/components/WelcomePage.web.js

```diff
@@ -343,12 +343,14 @@ class WelcomePage extends AbstractWelcomePage {
      * @returns {ReactElement}
      */
     _renderFooter() {
-        const { t } = this.props;
         const {
-            MOBILE_DOWNLOAD_LINK_ANDROID,
-            MOBILE_DOWNLOAD_LINK_F_DROID,
```

Comment:
Why are you not moving the f-droid link?

Reply:
moved as fDroidUrl under `android`